src/layout/HeroSection/HeroSection.svelte

```diff
@@ -50,7 +50,7 @@
 
 	onMount(async () => {
 		// Get the user's download preference
-		if (!localStorage.getItem("downloadSource")) {
+		if (!downloadSources.includes(localStorage.getItem("downloadSource") as DownloadSource)) {
```

Comment:
```suggestion
		if (!downloadSources.includes((localStorage.getItem("downloadSource") ?? "") as DownloadSource)) {
```

Reply:
I assume that's some typescript requirement I'm not aware of as I rarely use TS